tests/excel/test_range_specification.py

```
@@ -0,0 +1,234 @@
+            ["100", "200", "300", "400", "500", "600"],  # Row 6
+        ]
+
+        df = pd.DataFrame(data)
+
+        with pd.ExcelWriter(file_path, engine="openpyxl") as writer:
+            df.to_excel(writer, sheet_name="Sheet1", index=False, header=False)
+
+        return file_path
+
+    def test_basic_range_specification(self):
+        """基本的な範囲指定のテスト（未実装機能なので失敗する）。"""
```

Ruff (0.11.9)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)